Sources.txt

```diff
@@ -12,7 +12,7 @@ git;https://github.com/abbyyProduct/ThirdParty-MKL
 get;ThirdParty/MKL;/;MKL 2022.0.2 1.0.8.0
 
 git;https://github.com/abbyyProduct/ThirdParty-protobuf
-get;ThirdParty/protobuf;/;v3.11.4
+get;ThirdParty/protobuf;/;Golikov/v28
```

Comment:
Is it okay here to leave internal branch in master?

Reply:
I prepared a PR in `ThirdParty/protobuf` to merge, waiting all OCR tests succeeds
Yes, you right, I can already  merge it, the tag guarantees the required version